src/client/constants/constants.js

```diff
@@ -1,6 +1,17 @@
-const COLOR = {
+/* action types */
```

Comment:
`ACTIONS` are actions **types** in essence, it's a common approach to locate them next to the actions/action creators/thunks. So it'd be better to move they to `src/client/actions/types.js` for example.

Reply:
fixed